docs/serverless/rules/detections-logsdb-impact.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[detections-logsdb-index-mode-impact]]
+= Using logsDB index mode with {sec-serverless}
+
+LogsDB is enabled by default for Elastic serverless. This topic explains the impact of using logsDB index mode with  {sec-serverless}.  
```

Comment:
We should use the attribute [here](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc#L184). 
```suggestion
LogsDB is enabled by default for {serverless-full}. This topic explains the impact of using logsDB index mode with  {sec-serverless}.  
```